google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -864,3 +872,28 @@ def do_execute_no_params(self, cursor, statement, context=None):
         }
         with trace_call("SpannerSqlAlchemy.ExecuteNoParams", trace_attributes):
             cursor.execute(statement)
+
+
+# Alembic ALTER operation override
+@compiles(ColumnNullable, "spanner")
+def visit_column_nullable(
+    element: "ColumnNullable", compiler: "SpannerDDLCompiler", **kw
+) -> str:
+    return "%s %s %s %s" % (
+        alter_table(compiler, element.table_name, element.schema),
+        alter_column(compiler, element.column_name),
+        format_type(compiler, element.existing_type),
+        "" if element.nullable else "NOT NULL",
+    )
+
+
+# Alembic ALTER operation override
+@compiles(ColumnType, "spanner")
+def visit_column_type(
+    element: "ColumnType", compiler: "SpannerDDLCompiler", **kw
+) -> str:
+    return "%s %s %s" % (
+        alter_table(compiler, element.table_name, element.schema),
+        alter_column(compiler, element.column_name),
+        "%s" % format_type(compiler, element.type_),
+    )
```

Comment:
There are two syntax diverges:
Spanner supports `NULL/NOT NULL` statement without the `DROP` word in front of it for constraints dropping
Spanner doesn't use `TYPE` word for column type mentions

These two overrides covers both items.